pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -157,12 +157,13 @@ func (k *Bootstrapper) StartCluster(k8s config.KubernetesConfig) error {
 		return errors.Wrap(err, "parsing kubernetes version")
 	}
 
-	extraFlags := bsutil.CreateFlagsFromExtraArgs(k8s.ExtraOptions)
 	r, err := cruntime.New(cruntime.Config{Type: k8s.ContainerRuntime})
 	if err != nil {
 		return err
 	}
 
+	extraFlags := bsutil.CreateFlagsFromExtraArgs(k8s.ExtraOptions)
```

Comment:
this change needs to be added to the kicbs too
sorry for the inconvinence. the plan is to merge kicbs and kubeadm into one soon.

Reply:
I actually think this change was unnecessary, so I will remove it (for both)